drake/systems/Simulation.h

```diff
@@ -4,6 +4,12 @@
 #include <chrono>
 #include <stdexcept>
 #include <thread>
+#include <iostream>
+#include <Eigen/Dense>
+
+#include <drake/core/Vector.h>
+
+using Drake::toEigen;
```

Comment:
Headers should not have `using` at global scope; it pollutes the global namespace.